show/muxcable.py

```diff
@@ -2177,7 +2177,7 @@ def get_grpc_cached_version_mux_direction_per_port(db, port):
     mux_info_full_dict[asic_index] = state_db[asic_index].get_all(
         state_db[asic_index].STATE_DB, 'MUX_CABLE_INFO|{}'.format(port))
     trans_info_full_dict[asic_index] = state_db[asic_index].get_all(
-        state_db[asic_index].STATE_DB, 'TRANSCEIVER_STATUS|{}'.format(port))
+        state_db[asic_index].STATE_DB, 'TRANSCEIVER_INFO|{}'.format(port)) or {}
```

Comment:
what is the or condition here ?

Reply:
@vdahiya12 The or condition is just to ensure that an empty dictionary is returned rather None to avoid a crash.